autoupdate.go

```diff
@@ -9,10 +9,11 @@ import (
 	"strings"
```

Comment:
the only changes I made to this file are formatting and linting related, namely to shut my IDE up lmao.